docs/monoDrive_home/Weather.md

```diff
@@ -33,29 +33,12 @@ the custom map and it should now appear in the "World Outliner".
   <img class="lg_img" src="../img/weather_search.png" />
 </p>
 
-The weather also references the wind actor and the fog actor. Simply add these
-two actors anywhere in the map the same as the weather actor.
+The weather actor requires several other weather related actors to function. Place the following additional actors just as you placed the weather actor:
```

Comment:
suggestion: removed you and just
The weather actor requires several other weather related actors to function. Place the following additional actors similarly to the weather actor: